### DISCUSSION TITLE
Unable to apply patch

### DISCUSSION THREAD

**Discussion by MrSuddenJoy** [symfony/symfony#58817] [Q&A]
### Symfony version(s) affected

Latest

### Description

Am developing our inhouse software that heavily relies on `Symfony`. `Dependabot` warned me that there is an issue in `symfony` that needs to be patched. While trying to apply [this very patch](https://github.com/symfony/symfony/commit/18ecd03eda3917fdf901a48e72518f911c64a1c9) my project fails on building phase. Whats strange, when I revert commit with patch applied, project builds with success. This makes me think there is something going on with this patch.

May I ask for assistance?

### How to reproduce

You can try to apply this patch and wait.

### Possible Solution

No idea. Am not familiar with Symfony codebase.

**Comment by stof** [comment]
Without telling us which error you get, there is nothing we can do to help you.

**Comment by MrSuddenJoy** [comment]
@stof here you go: screengrab from `Actions` tab
![Zrzut ekranu 2024-11-8 o 20 14 32](https://github.com/user-attachments/assets/4e8cfbf9-1cd1-4e33-92ee-07397af721bd)

**Comment by stof** [comment]
"unknown error" is not really actionable (and I cannot click in your screenshot to get more information, and this would not help me anyway given it says the link requires write access to the repo anyway).

And this is an error in dependabot, not in Symfony.

**Comment by MrSuddenJoy** [comment]
> And this is an error in dependabot, not in Symfony.

@stof will take indepth look into it and report back.
